### DISCUSSION TITLE
Control Center not showing in Stream Deck

### DISCUSSION THREAD

**Discussion by clasener** [elgatosf/elgato#87] [Help] [answered]
Hi. I've got two keylights connected through Control Center, but the Stream Deck does not show the Control Center Plugin. I've uninstalled everything and tried again.. still it does not work.

Situation: MacBook Pro, M1, MacOs Sonoma (latest)

Please help... The stream deck is less fun without this plugin :)

**Comment by tiptronic** [comment]
HI @clasener - did you install the ControlCenter plugin from the store? (It is NOT automatically installed)

**Reply by clasener** [reply]
<img width="844" alt="Screenshot 2023-12-08 at 10 41 47" src="https://github.com/elgatosf/elgato/assets/51059903/d33fb02a-e32e-4419-9fb0-1906590d3b77">

Yes. It was downloaded there and installed it. It does run on it's own, but not within the Stream Deck app.

Thanks for the swift reply by the way!

**Comment by tiptronic** [comment, marked as answer]
Hi @clasener 
no - you don't have the plugin installed, but the Control-Center.app...

Just visit  [https://marketplace.elgato.com/stream-deck/plugins](https://marketplace.elgato.com/stream-deck/plugins)
by clicking the 'Marketplace' icon in the Stream Deck window and look for `Control Center` in the plugins section...

This plugin is needed to talk to the Control-Center.app...

**Reply by clasener** [reply]
You are such a big help! I did do that though... but because you've mentioned it too... did it again. Didn't work. Then I did it again with the plugin panel open and it popped right in there! Thanks so much!
